source/triggers/aws-eventbridge.txt

```diff
@@ -152,6 +164,238 @@ Procedure
+                     "private": <boolean>
+                  }
+               ]
+
+            For more information on function configuration files, 
+            see :ref:`Function Configuration Files <appconfig-functions>`.
+
+         .. step:: Add an Error Handler to Your Trigger Configuration
+
+            Add an ``error_handler`` attribute to your trigger configuration file
+            in the ``Triggers`` folder. The trigger configuration file should 
+            resemble the following:
+
+            .. code-block:: json
+
+                     {
+                     "name": "...",
+                     "type": "...",
+                     "event_processors": {
+                        "AWS_EVENTBRIDGE": {
+                              "config": {
+                                 "account_id": "<AWS Account ID>",
+                                 "region": "<AWS Region>",
+                                 "extended_json_enabled": <boolean>
+                              }
+                           }
+                        },
+                        "error_handler": {
+                           "config": {
+                              "enabled": <boolean>,
+                              "function_name": "<Error Handler Function Name>"
+                           }
+                        }
+                     }
+            
+            For more information on trigger configuration files, see
+            :ref:`Trigger Configuration Files <appconfig-triggers>`.
+
+.. _eventbridge-error-handler-parameters:
+
+Error Handler Parameters
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default error handler has two parameters: ``error`` and ``changeEvent``.
+
+``error`` 
+`````````
+Has two attributes: ``code`` and ``message``.
+
+- ``code``: For a list of error codes 
+used by the error handler, see the below section on :ref:`Error 
+Codes <eventbridge-error-codes>`. 
+
+- ``message``: The unfiltered error message from an errored Eventbridge
+put request.
+
+``changeEvent``
+```````````````
+
+The requested change to your data made by EventBridge. For more information
+on types of change events and their configurations, see 
+:manual:`Change Events </reference/change-events>`.
+
+.. _eventbridge-error-codes:
+
+Error Codes
+~~~~~~~~~~~
+
+If an error was recevied from EventBridge, the event processor will parse the 
+error as either ``DOCUMENT_TOO_LARGE`` or ``OTHER``. This parsed error is passed
+to the error handle function. 
+
+``DOCUMENT_TOO_LARGE``
+``````````````````````
+If the put entry for an EventBridge trigger event is larger
+than 256 KB, Eventbridge will throw an error. The error will either contain:
+
+1.  `status code: 400 <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400>`_ and 
+``total size of the entries in the request is over the limit``.
+
+2. `status code: 413 <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413>`_, 
+which indicates a too large payload. 
```

Comment:
there isn't a corresponding message like the above status code?

Reply:
found this message in the spec, but can't find anything online. Will ask Taylor

Reply:
@tcannon91 Is there a resource online from AWS or otherwise about this error message that I can link? Found this error message in the spec doc.

Reply:
No, there isn't a consistent message in the response from aws when we get a 413 status code -- there might not be any specific error, just a status code.

I think it is fine to have this documentation but it might be worth calling out that we are trying to use the error code to abstract this information away from users -- like this list could shift slightly or if there are additional errors that AWS introduces/starts returning that are just variations on document too large errors. I only bring it up in case we are concerned with this list getting stale. But it is accurate for what we are using/checking today and it isn't secret information so no issues with it being in the docs :)